### ISSUE TITLE
Provide inverse gamma prior with indeterminate moments

### ISSUE THREAD

**Issue by JohannesPfeifer** [DynareTeam/dynare#846]
We currently only allow specifying inverse gamma priors with finite/unique mean and variance. But Leeper/Walker/Yang (2010) in an influential paper use an inverse gamma prior with s=1 and nu=4 (parametrization as at http://en.wikipedia.org/wiki/Inverse-gamma_distribution), which implies non-existing first and second moments. I would suggest to provide a new prior ```inv_gamma_parametrized``` that takes the values provided in the mean and standard deviation fields of the ```estimated_params```-block directly as the parameters of the distribution, thereby avoiding the impossible transformation from mean and variances. 
I would implement this when doing #520

**Comment by MichelJuillard** [comment]
I would permit it only with the new interface for prior declaration described here: http://www.dynare.org/DynareWiki/NewEstimation
In the old interface, I don't like the idea to use parameter 1 and 2 for anything else but mean and variance.
We need to check if the code for the distribution allows for case like s=1 nu=4

**Comment by JohannesPfeifer** [comment]
How for out is the milestone for the new estimation interface? I see your point about the first two parameters being reserved for mean and standard deviations. That is why my proposal was to use a very explicit name for the distribution that makes clear we parameterize it using the shape and location parameter.

The current code for the distribution would already allow for this. For example, the referenced case above in our parameterization is equal to
```
s=8;
nu=2;
x=(0:0.1:8);
dens = exp(lpdfig2(x,s,nu));  
plot(x,dens);
```
But currently there is no way to get this specification.

**Comment by stepan-a** [comment]
I agree with @MichelJuillard , this will have to wait for the new estimation interface (this summer).
